### PR TITLE
[docs] Update front page with feedback from marketing team

### PR DESCRIPTION
**Pull request by andrewnicols**
This is a WIP, but so far uncorporates:

- Adopt .com-style navbar
- Switch logo (need one with TM apparently but have not got this yet)
- switch to cards (awaiting images or mockups for desired look)
- switch title to new style with em
- ditch orange hero banner
- switch font to Open Sans for most content
- switch front to DM Sans Serif for hero banner titles

Hopefully we'll have some copy, images and final mockups soon.

If we can get the copy sooner then I'll land this change as-is and we can sort out the other pieces later.

<a href="https://gitpod.io/#https://github.com/moodle/devdocs/pull/164"></a>

### CODE REVIEW

File: src/components/HomepageFeatures/index.js [hunk 1]
```diff
@@ -42,34 +42,54 @@ const FeatureList = [
...
+        title: 'Release notes',
         Svg: require('@site/static/img/undraw_docusaurus_react.svg').default,
         description: (
             <>
-                Discover our community, and learn about our mission, and our roadmap.
+                Technical specs and new features of Moodle releases, supported versions and more.
             </>
         ),
-        linkText: 'Discover Communities',
+        linkText: 'Check the Release notes',
```
Comment (sarjona) [inline src/components/HomepageFeatures/index.js:81]:
```suggestion
        linkText: 'Check the release notes',
```

File: src/components/HomepageFeatures/index.js [hunk 2]
```diff
@@ -42,34 +42,54 @@ const FeatureList = [
         Svg: require('@site/static/img/undraw_docusaurus_tree.svg').default,
         description: (
             <>
-                Learn about our coding standards, styles, and conventions.
+                Read our style guidelines, including coding standards and naming conventions.
             </>
         ),
-        linkText: 'Learn more',
+        linkText: 'View Standards',
```
Comment (sarjona) [inline src/components/HomepageFeatures/index.js:48]:
```suggestion
        linkText: 'View standards',
```

**Review by sarjona — CHANGES_REQUESTED** [review]
The patch looks good. Thanks for working on it and making it seems like moodle.com. Well done! :-)

As you'll see, there are only a minor uppercases that I think should be moved to lowercase.

Apart from that, I hope we receive the design soon. Although the current one looks more similar to moodle.com, I'm still not convinced with the colours and I think we need a different logo, a mark or anything to differentiate moodle.com from Moodle devdocs.

File: src/components/HomepageFeatures/index.js (hunk 1 above)
Comment (andrewnicols) [reply src/components/HomepageFeatures/index.js:81]: I disagree with this one. "Release notes" are a proper noun, they're not just another word. They should therefore be capitalised.

Comment (andrewnicols) [reply src/components/HomepageFeatures/index.js]: Done.

**Pull request merged**